docs/source/format/CanonicalExtensions.rst

```diff
@@ -51,7 +51,7 @@ types:
 
   3) Its serialization *must* be described in the proposal and should
      not require unduly implementation work or unusual software dependencies
-     (for example, a trivial custom text format or JSON would be acceptable).
+     (for example, a trivial custom text format or YAML would be acceptable).
```

Comment:
@ianmcook is giving another example ok?

Reply:
This sentence is **not** an example of a canonical extension type. It describes how canonical extension types _may_ be serialized as metadata.

So this should remain JSON. We would definitely not allow YAML as the serialization format for canonical extension types.

Reply:
@ianmcook 's suggestion of "a JSON-based format" is correct though.

Reply:
Got it. Switched to @ianmcook's suggestion.